README.md

```diff
@@ -8,9 +8,16 @@ Contains the source code for the thesis completed under Dr. Philip B. Yasskin
 
 ```shell
 docker-compose -f docker-compose-dev.yml run users python manage.py # list commands for users flask cli
-docker-compose -f docker-compose-dev.yml run users python manage.py seed-db # example of cli command (seed the db)
+docker-compose -f docker-compose-dev.yml run users python manage.py seed-db # example of cli command (seed the user db)
+docker-compose -f docker-compose-dev.yml run users flake8 project # code quality with flake8 for users
+docker-compose -f docker-compose-dev.yml run exercises python manage.py # list commands for exercises flask cli
+docker-compose -f docker-compose-dev.yml run exercises python manage.py seed-db # example of cli command (seed the exercises db)
+docker-compose -f docker-compose-dev.yml run exercises flake8 project # code quality with flake8 for project
 docker-compose -f docker-compose-dev.yml build # build
 docker-compose -f docker-compose-dev.yml up -d # bring up
 docker-compose -f docker-compose-dev.yml logs # view logs
-docker-compose -f docker-compose-dev.yml run users flake8 project # code quality with flake8
 ```
+
+## Documentation
+
+* [Neo4j Python Driver](https://neo4j.com/docs/api/python-driver/current/)
```

Comment:
what?

Reply:
Bookmarks to documentation!

Reply:
I don't think that is readme worthy. Add it to the drive under resources I guess if you really want

Reply:
SAD! Removed in `bc9007`

Reply:
https://github.com/JosephMart/exercise-dependency-web/pull/2/commits/bc9a007e27fb10056d02bb1ec2c472120b27401a